### PR TITLE
Added 6.5 release notes

### PR DESCRIPTION
**Pull request by gmoyerbrailean**

### CODE REVIEW

File: _docs/release-notes/basespace/2021/6.5.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Features
+ - Updated the look and feel of the Settings pages.
+ - Improved behavior when creating a new Project in a Project Chooser modal. Creating a new Project automatically selects the and closes the modal.
+ - Improved Project support when using the V2 Sample Sheet format. The optional ProjectName column is now used for Biosample Default Project and Output Project, when specified.
+
+### Bugs Fixed
+ - Fixed an issue that could prevent appsession logs from being displayed.
+ - Fixed an issue on app input forms that caused the Add a New Row button to be invisible in Safari.
+ - Fixed an issue on the Run Metris tab where the Tbp unit was not being used for yield values above 1000 Gbp.
```
Comment (whedley) [inline _docs/release-notes/basespace/2021/6.5.0.md:21]: @gmoyerbrailean  Typo. Metrics

**Review by whedley — CHANGES_REQUESTED** [review]

File: _docs/release-notes/basespace/2021/6.5.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+tags: [basespace]
+version: 6.5.0
+release_date: 2021-01-19
+---
+
+## What's changed for BaseSpace Sequence Hub 6.5
+
+### Features
+ - Updated the look and feel of the Settings pages.
+ - Improved behavior when creating a new Project in a Project Chooser modal. Creating a new Project automatically selects the and closes the modal.
```
Comment (whedley) [inline _docs/release-notes/basespace/2021/6.5.0.md:15]: Grammar. Selects **it** ?

Comment (harmonj3) [reply _docs/release-notes/basespace/2021/6.5.0.md:15]: Just delete 'the'

**Review by harmonj3 — APPROVED** [review]
lgtm

**Review by harmonj3 — APPROVED** [review]
lgtm

**Review by whedley — APPROVED** [review]
TLTQ

**Comment by gmoyerbrailean** [comment]
Thanks @whedley good catches! Fixed.

@harmonj3 after long deliberation I decided to implement a similar fix to your suggestion. Hopefully it's okay.

**Pull request merged**